src/components/modal/v2/styles/components/_disclaimer.scss

```
@@ -0,0 +1,127 @@
+.disclaimer {
+    text-align: left;
+    box-sizing: border-box;
+
+    @include desktop {
+        padding-left: 30px;
+    }
+
+    @include tablet {
+        width: 100%;
+        justify-content: center;
+    }
+
+    ul {
+        list-style: disc;
+        @include desktop {
+            padding: 0px;
+        }
+
+        @include mobile {
+            padding: 0px;
+        }
+        &.ppc {
+            @include mobile {
+                margin-top: -11px;
+                margin-bottom: 0px;
+            }
+        }
+        .disclaimer__item-wrapper {
+            display: list-item;
```

```suggestion
            display: list-item;
            padding-bottom: 0px;
```
Looks like there was a little bit of extra padding between the preapproval disclaimer items that we don't need.